.circleci/config.yml

```diff
@@ -432,7 +432,7 @@ jobs:
               exit 0;
             fi
 
-            SEGMENT_WRITE_KEY=$SEGMENT_WRITE_KEY_STAGE yarn package:<< parameters.env >>
+            UPGRADES_LINK=UPGRADES_LINK_STAGE SEGMENT_WRITE_KEY=$SEGMENT_WRITE_KEY_STAGE yarn package:<< parameters.env >>
```

Comment:
I think circle ci env variable should start with symbol `$`, isn't it?
`UPGRADES_LINK=$UPGRADES_LINK_STAGE`

Reply:
Good catch! Thanks!